src/mapport.cpp

```diff
@@ -153,44 +153,23 @@ void StartThreadMapPort()
 
 static void DispatchMapPort()
```

Comment:
DispatchMapPort is local and only called in one place; might as well roll it into `StartMapPort`?

Reply:
Done.